templates/taxi/car_list.html

```diff
@@ -1,16 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
```

Comment:
The inline style for the 'Create' link is used to float it to the right. Consider using a CSS class instead of inline styles for better maintainability and separation of concerns.